ls1intum/Artemis
# `Exam mode`: Fix text exercise feedback view

### Checklist
#### General
#### Server

### Motivation and Context
Fixes #7111

### Description
The client checks the exam publication date to determine if the result should be shown to the student. This information was missing from the server response.

### Steps for Testing
Prerequisites:
- 1 Exam with a corrected text exericse
- 1 Student

1. View the summary and press view result for the text exercise
2. Check that feedback is visible

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests
